pkg/projects/Microsoft.NETCore.Console/project.json

```diff
@@ -0,0 +1,51 @@
+{
+  "dependencies": {
+    "Microsoft.NETCore": "5.0.1-rc3-24126-00",
+    "Microsoft.NETCore.ConsoleHost": "1.0.0-rc3-24126-00",
+    "Microsoft.NETCore.Runtime.CoreCLR": "1.0.2-rc3-24126-00",
```

Comment:
IIRC, the Microsoft.NetCore.App metapackage was referencing a "NetCore.Runtimes" metapackage that, in turn, pulled in CoreCLR. Here, we are pulling in CoreCLR package directly. Is this difference intentional or should we strive for consistency?